lib/sqlalchemy/dialects/mysql/mariadb.py

```diff
@@ -7,11 +7,20 @@
 # mypy: ignore-errors
 from .base import MariaDBIdentifierPreparer
 from .base import MySQLDialect
+from .base import MySQLTypeCompiler
 from ... import util
+from ...sql import sqltypes
 from ...sql.sqltypes import UUID
 from ...sql.sqltypes import Uuid
 
 
+class INET6(sqltypes.TypeEngine[str]):
+    """
+    INET6 column type for MariaDB
+    """
```

Comment:
**Michael Bayer** (zzzeek) wrote:

needs .. versionadded::

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5598

Reply:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5598